zap/src/main/java/org/zaproxy/zap/extension/ascan/ActiveScanPanel.java

```diff
@@ -66,17 +72,23 @@
     private static final String ZERO_NEW_ALERTS_LABEL_TEXT = "0";
 
     private static final ActiveScanTableModel EMPTY_RESULTS_MODEL = new ActiveScanTableModel();
+    private static final FilterMessageTableModel EMPTY_FILTER_MESSAGE_MODEL =
+            new FilterMessageTableModel();
 
     private ExtensionActiveScan extension;
     private JScrollPane jScrollPane;
+    private JScrollPane filterPane;
```

Comment:
This can be removed (and `jScrollPane`, now only used in the `getWorkPanel`).

Reply:
you are saying this because of mainpanel which is not null so getWorkpanel will not even again initialise filterpane so no need to make it instance variable ?

Reply:
Yes.